cmd/doctext/main.go

```diff
@@ -78,11 +84,14 @@ func main() {
 	}
 	wg.Wait()
 
-	for assignee, issue := range issues {
-		teamMember, ok := team[assignee]
-		if !ok {
-			teamMember = team["team"]
+	for assigneeJiraName, issue := range issues {
+		var teamMember TeamMember
+		if person, ok := team.PersonByJiraName(people, assigneeJiraName); ok {
+			teamMember = TeamMember{SlackId: person.Slack}
+		} else {
+			teamMember = TeamMember{SlackId: "!subteam^SKW6QC31Q"}
```

Comment:
Can we make this a constant? It would be good also to keep the comment explaining how to retrieve the team ID.

Reply:
Good idea! Done, PTAL :)

Reply:
Didn't you want to keep the code comment explaining how to retrieve the group ID?